chihuahua/assetlist.json

```diff
@@ -308,6 +308,52 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/migaloo/images/ampgash.png"
       }
+    },
+    {
+     "denom_units": [
+        {
+          "denom": "ibc/DBBDE1753EFA93F09C468DB5E384861BB741D4AA73E9C7032D139BB2C99FDD65",
+          "exponent": 0,
+          "aliases": [
+            "factory/migaloo1etlu2h30tjvv8rfa4fwdc43c92f6ul5w9acxzk/uguppy"
+          ]
+        },
+        {
+          "denom": "GUPPY",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/DBBDE1753EFA93F09C468DB5E384861BB741D4AA73E9C7032D139BB2C99FDD65",
+      "name": "GUPPY",
+      "display": "GUPPY",
+      "symbol": "GUPPY",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "migaloo",
+            "base_denom": "factory/migaloo1etlu2h30tjvv8rfa4fwdc43c92f6ul5w9acxzk/uguppy",
+            "channel_id": "channel-39"
+          },
+          "chain": {
+            "channel_id": "channel-10",
+            "path": "transfer/channel-10/factory/migaloo1etlu2h30tjvv8rfa4fwdc43c92f6ul5w9acxzk/uguppy"
```

Comment:
The channels are backwards and the sha256 hash of the path does not equal DBB...D65. Reversing the channels will fix it.